config/zones.json

```
@@ -3778,16 +3778,16 @@
       [126.537423944, 18.5052273625]
     ],
     "capacity": {
-      "battery storage": 11,
-      "biomass": 483,
-      "coal": 10944,
+      "battery storage": 34,
+      "biomass": 501,
+      "coal": 11684,
       "gas": 3453,
-      "geothermal": 1928,
-      "hydro": 3031,
-      "hydro storage": 748,
+      "geothermal": 1932,
+      "hydro": 3035,
+      "hydro storage": 736,
```

These add up to 3771 MW but the PDF has a installed capacity of 3781 MW so there is 10 MW missing somewhere. My bet would be hydro storage since it changed by -12 MW but please double check.

---

@VIKTORVAV99

I messed up with my calculation, the hydro capacity should be 3045 instead of 3035. I would like to reflect the following changes:

```suggestion
      "hydro": 3045,
      "hydro storage": 736,
```

Anyways, please see attached reference for the hydro storage capacity.

Find: **Kalayaan PSPP**
[[01]_2022_loepp_grid-connected_luzon_june-2022.pdf](https://github.com/electricitymap/electricitymap-contrib/files/9333044/01._2022_loepp_grid-connected_luzon_june-2022.pdf).